libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -3098,6 +3012,122 @@ string YulUtilFunctions::conversionFunction(Type const& _from, Type const& _to)
 	});
 }
 
+string YulUtilFunctions::copyStructToStorageFunction(StructType const& _from, StructType const& _to)
```

Comment:
Any changes here?

Reply:
It is a plain copy/paste of the `StructType` case in `updateStorageValueFunction`